elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -39,6 +39,15 @@ See mixin blocks before selectors that contain the selector name and the three l
   display: none;
 }
 
+:host(.pfe-sticky) {
+  position: sticky;
```

Comment:
Change this to fixed if we're going with a JS solution for this (we could also try just using `position: sticky` instead with no JS and event listener, but that wouldn't work on IE, and sticky has bugs.)

Reply:
@wesruv the JS and position sticky is in place to work with banners when they are above the navigation. I didn't think we could replicate that feature using position fixed (at least not very easily) so I went with the solution already used in the original nav.

Reply:
Position sticky does what your JS scroll & sticky handler does, but with bugs. If position sticky was better supported **wasn't** a buggy mess, I'd say just use `sticky` in the CSS, and no JS.

Have you tested with this line being changed to fixed? It should work exactly the same, but with less browser bugs.

Reply:
@wesruv ahh I understand now, thanks for explaining further! When I tested initially I missed some more CSS tweaks I needed to make using position fixed for everything to look good which made me concerned I could not replicate the sticky behavior with fixed.  I'll get those changes made.

Reply:
@wesruv added those changes.